issue_template.md

```diff
@@ -1,18 +1,18 @@
-**Description:**
-<!-- Give a brief description of the issue -->

```

Comment:
Why did you commit this file? I think we do not need to do any changes to this file

Reply:
Fixed with dbcfae187512aa0491af43140d1faf2b9612b259